Description: Re-enable plugin loading in qpsmtpd-forkserver to fix pre-connection hooks
Author: Andreas Erhard <erhard@wor.net>
Origin: upstream, https://github.com/smtpd/qpsmtpd/issues/308
Bug: https://github.com/smtpd/qpsmtpd/issues/308
Last-Update: 2023-09-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: qpsmtpd/qpsmtpd-forkserver
===================================================================
--- qpsmtpd.orig/qpsmtpd-forkserver
+++ qpsmtpd/qpsmtpd-forkserver
@@ -193,7 +193,7 @@ POSIX::setgid($qgid) or die "unable to c
 POSIX::setuid($quid) or die "unable to change uid: $!\n";
 $> = $quid;
 
-#$qpsmtpd->load_plugins;
+$qpsmtpd->load_plugins;
 
 foreach my $addr (@LISTENADDR) {
     ::log(LOGINFO, "Listening on $addr->{addr}:$addr->{port}");
